README.md

```diff
@@ -36,11 +35,17 @@ You can now claim the initial NEO and GAS:
 
 `./create_wallet` will display several internal error messages, which is expected as long as at the end you still get a success message.
 
+If you call `./create_wallet.sh` or `./docker_run_and_create_wallet.sh`, it will create 2 files in your current directory :
```

Comment:
remove the space before the colon here plase.

Reply:
I will, hope you didn't mind the extra work, will do better next time =)